app/components/number-widget/component.js

```diff
@@ -7,6 +7,7 @@ export default Ember.Component.extend({
     settings : {
         fontSize: 3,
         fontColor: '#F44336',
+        noLink: false,
```

Comment:
Although technically the status should be isLink: true in the default settings, but I think the problem is we are not extending the settings but replacing them. Probably best practice is to set state in a descriptive way like "isLink" and extend the settings object.